content/webapp/pages/landing-page.js

```diff
@@ -58,20 +58,9 @@ export class Page extends Component<Props> {
       ) : null
     ) : null;
 
-    // TODO: This is not the way to do site sections
-    const breadcrumbs = {
```

Comment:
breadcrumb items have an `isHidden` property, and I initially thought about storing the `id`s of pages we wouldn't want to show up in an array and hide the breadcrumb accordingly, but since this is a `landing-page`, it seemed like unnecessary over-complication.